tests/console/system_prepare.pm

```diff
@@ -55,7 +55,11 @@ sub run {
             add_suseconnect_product(get_addon_fullname('desktop'), undef, undef, undef,                             300, 1);
             add_suseconnect_product(get_addon_fullname('we'),      undef, undef, "-r " . get_var('SCC_REGCODE_WE'), 300, 1);
         }
-        my $myaddons = get_var('SCC_ADDONS');
+        my $myaddons = get_var('SCC_ADDONS', '');
+        $myaddons .= "dev,lgm,wsm" if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
```

Comment:
if there is a SDK in SCC_ADDONS, you haven't deal with such situation.

Reply:
sdk       => is_sle('15+') ? 'sle-module-development-tools' : 'sle-sdk',
It did register at lib/registration.pm file